Updateed severity classification section in vulnerability-management-process (#64)
wso2/docs-security: en/docs/security-processes/vulnerability-management-process.md

### Severity Classification

> We follow the [Common Vulnerability Scoring System v3.1](https://www.first.org/cvss/specification-document) to rate the vulnerabilities that are reported to us and following this process only to products within [wso2 support matrix](https://wso2.com/products/support-matrix/).

Reviewer: ```suggestion
We follow the [Common Vulnerability Scoring System v3.1](https://www.first.org/cvss/specification-document) to rate the vulnerabilities that are reported to us. This process is followed for supported products listed within the [WSO2 Support Matrix](https://wso2.com/products/support-matrix/).
```
